src/Illuminate/Queue/Queue.php

```diff
@@ -188,4 +195,27 @@ public function setContainer(Container $container)
     {
         $this->container = $container;
     }
+
+    /**
+     * Set the queue prefix.
+     *
+     * @param  string  $prefix
+     * @return $this
+     */
+    public function setQueuePrefix($prefix = null)
```

Comment:
@browner12 Why are these not called `setPrefix` and `getPrefix`? After all, we're already in a `Queue` class here...